client/src/components/RelatedItems/YourOutfit.jsx

```diff
@@ -1,4 +1,5 @@
 import React, { useContext } from 'react';
+import _ from 'underscore';
```

Comment:
it looks like underscore is not used in this file. Is it included here for future-proofing?